modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb

```diff
@@ -431,5 +433,11 @@ def statsd_tags
     def error_notification?
       notification_type == :error
     end
+
+    def needs_confirmation_number?(config)
```

Comment:
It seems like the logic here should be an && for the `config[:notification_type] != 'DUPLICATE'`

For example If `config[:notification_type]` is `'REJECTED'`, 
then `'REJECTED' != 'REJECTED'` is `false`, 
but `'REJECTED' != 'DUPLICATE'` is true....making the overall condition = true.

I think it could be written like this then

  `config[:form_number] != 'vba_26_4555' 
  && !['REJECTED', 'DUPLICATE'].include?(config[:notification_type])`

Reply:
Thanks! I've updated my PR.